### DISCUSSION TITLE
Keyboard shortcut list

### DISCUSSION THREAD

**Discussion by xyzzy-plugh-plover** [HexFiend/HexFiend#431] [Q&A]
Is there a HexFiend keyboard shortcut list?

I specifically want to reload a binary template via the keyboard instead of via the mouse.

Thanks.

**Comment by wader** [comment]
Hi, i've documented how to trigger reload using osascript here https://github.com/HexFiend/HexFiend/blob/master/templates/Tutorial.md#automatically-reload-template-on-save

If you wan't automatic reload i think it should be done automatically now since https://github.com/HexFiend/HexFiend/commit/4128c583c526dc91be864055dad1fe638100a711

Hope that helps

**Reply by xyzzy-plugh-plover** [reply]
Templates already reload on save for me on 2.16, but I prefer to use read-only mode, which doesn't allow me to save.

If a distinct keyboard command to reload a template doesn't exist, I'd definitely appreciate one being added.
